databricks/sdk/core.py

```diff
@@ -91,12 +91,25 @@ def inner() -> Dict[str, str]:
 @credentials_provider('runtime', [])
 def runtime_native_auth(cfg: 'Config') -> Optional[HeaderFactory]:
     from databricks.sdk.runtime import init_runtime_native_auth
-    if init_runtime_native_auth is None:
-        return None
-    else:
+    if init_runtime_native_auth is not None:
         host, inner = init_runtime_native_auth()
         cfg.host = host
         return inner
+    try:
+        from dbruntime.databricks_repl_context import get_context
+        ctx = get_context()
```

Comment:
Just want to double check when is this called? Inside a command or during REPL initialization?

Reply:
both